MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -3690,6 +3691,16 @@ public void removeForce(Force force) {
         if (null != force.getParentForce()) {
             force.getParentForce().removeSubForce(fid);
         }
+        
+        // clear out StratCon force assignments
+        for (AtBContract contract : getActiveAtBContracts()) {
```

Comment:
In the future we should probably add a getActiveStratConContracts method to campaign